### PR TITLE
#256 Explicitly instruct the user to name the cluster

### PR DESCRIPTION
**Pull request by ALRubinger**

### CODE REVIEW

**Comment by ALRubinger** [comment]
screenshot7

I signed it!

File: docs/hellonode.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ If all goes well, you should be able to see the container image listed in the co
 
 A cluster consists of a master API server and a set of worker VMs called nodes. 
 
-Create a cluster via the Console: *Compute > Container Engine > Container Clusters > New container cluster*. Leave all the options default, and you should get a Kubernetes cluster with three nodes, ready to receive your container image.
+Create a cluster via the Console: *Compute > Container Engine > Container Clusters > New container cluster*. Set the name to 'hello-world', leaving all other options default.  You should get a Kubernetes cluster with three nodes, ready to receive your container image..
```
Comment (johndmulhausen) [inline docs/hellonode.md]: You have an extra period at the end of the last sentence and and extra space before it. :)

**Comment by johndmulhausen** [comment]
ping @ALRubinger -- You're so close! :)

**Comment by pwittrock** [comment]
@ALRubinger is this still active?

**Comment by ALRubinger** [comment]
@pwittrock @johndmulhausen Updated the PR with a new commit, 07bd163, to remove the extra period; thanks!

**Comment by johndmulhausen** [comment]
Thanks a ton, @ALRubinger, and thanks for the review @pwittrock !

**Pull request merged**